app/src/main/java/co/develhope/chooseyourownbeer/network/BeersProvider.kt

```diff
@@ -0,0 +1,22 @@
+package co.develhope.chooseyourownbeer.network
+
+import co.develhope.chooseyourownbeer.ui.model.BeerUi
+import co.develhope.chooseyourownbeer.network.dto.toListOfBeerUi
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+const val PAGE_SIZE = 20
```

Comment:
in teoria questo non serve più perchè non lo usiamo all'interno del get del BeerService